[Bug report]重新安装脚手架后无法create项目

**问题描述**  
请用简洁的语言描述你遇到的bug，至少包括以下部分，如提供截图请尽量完整：
1. 问题触发的条件 npm i -g @mpxjs/cli，mpx create mpx-project
2. 期望的表现 创建项目
3. 实际的表现  Cannot find module '@vue/cli-shared-utils'然后无法创建

**环境信息描述**  
至少包含以下部分：
1. 系统类型(Mac或者Windows)window
2. Mpx依赖版本![image](https://github.com/didi/mpx/assets/61222573/3be9e94c-7896-4d2d-b4c5-3e67e40089d7)
3. 小程序开发者工具信息(小程序平台、开发者工具版本、基础库版本）
D:\uniapp> mpx create prject
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@vue/cli-shared-utils'
Require stack:
- C:\Users\24342\AppData\Roaming\npm\node_modules\@mpxjs\cli\utils\index.js
- C:\Users\24342\AppData\Roaming\npm\node_modules\@mpxjs\cli\bin\mpx.js

--- a/packages/mpx-cli/package.json
+++ b/packages/mpx-cli/package.json
@@ -6,6 +6,7 @@
   "dependencies": {
     "@types/inquirer": "^8.1.3",
     "@vue/cli": "^5.0.0",
+    "@vue/cli-shared-utils": "^5.0.4",
     "commander": "^7.1.0",
     "fs-extra": "^9.1.0",
     "inquirer": "^8.0.0",

--- a/packages/vue-cli-plugin-mpx/config/base.js
+++ b/packages/vue-cli-plugin-mpx/config/base.js
@@ -20,7 +20,10 @@ const { getReporter } = require('../utils/reporter')
  * @param { string } name
  */
 function changeStyleVueRuleToMpx (config, name) {
+  // 移除vue-modules规则，因为mpx在构建时会在resourceQuery携带filename，这会导致带有
+  // module命名的mpx文件命中vue-modules规则，导致样式编译错误
   config.module.rule(name).oneOfs.delete('vue-modules')
+  // 抹平web和mp配置的命名，方便一次修改
   const store = config.module.rule(name).oneOfs.store
   const value = store.get('vue')
   value.name = 'mpx'

--- a/packages/vue-cli-plugin-mpx/config/base.js
+++ b/packages/vue-cli-plugin-mpx/config/base.js
@@ -20,6 +20,7 @@ const { getReporter } = require('../utils/reporter')
  * @param { string } name
  */
 function changeStyleVueRuleToMpx (config, name) {
+  config.module.rule(name).oneOfs.delete('vue-modules')
   const store = config.module.rule(name).oneOfs.store
   const value = store.get('vue')
   value.name = 'mpx'

